assets/css/main.css

```diff
@@ -20,12 +20,12 @@ h1 {
 }
 
 .jumbotron {
-  background: #ff407c; /* For browsers that do not support gradients */
-  background: -webkit-radial-gradient(#915f6d, #915f6d); /* Safari 5.1 to 6.0 */
-  background: -o-radial-gradient(#915f6d, #915f6d); /* For Opera 11.6 to 12.0 */
-  background: -moz-radial-gradient(#915f6d, #915f6d); /* For Firefox 3.6 to 15 */
-  background: radial-gradient(#915f6d, #915f6d); /* Standard syntax */
-  color: #fff;
+  background: #ff407ce6; /* For browsers that do not support gradients */
```

Comment:
These issues were solved in PR #12 

Can you please rebase your branch with current master to have these changes?